google/cloud/pubsub/internal/tracing_batch_sink.cc

```diff
@@ -60,6 +60,11 @@ auto MakeParent(Links const& links, Spans const& message_spans,
                 pubsub::Topic const& topic) {
   namespace sc = ::opentelemetry::trace::SemanticConventions;
   opentelemetry::trace::StartSpanOptions options;
+  opentelemetry::context::Context root_context;
+  // TODO(#13287): Use the constant instead of the string.
+  root_context.SetValue(/*opentelemetry::trace::kIsRootSpanKey=*/"is_root_span",
+                        true);
```

Comment:
This has no effect in older versions of `opentelemetry-cpp`. so....

Reply:
Yeah... Can't really change that....